lib/config/fileconf.go

```diff
@@ -1519,6 +1519,8 @@ type GCPMatcher struct {
 type AccessGraphSync struct {
 	// AWS is the AWS configuration for the AccessGraph Sync service.
 	AWS []AccessGraphAWSSync `yaml:"aws,omitempty"`
+	// PollInterval is the frequency at which to poll for AWS resources
+	PollInterval time.Duration `yaml:"poll_interval,omitempty"`
```

Comment:
Should there be a default for this?

Reply:
Yes, I'll add the 15 minute default back.